app/qml/pages/DeviceListPage.qml

```diff
@@ -276,6 +276,10 @@ Page {
         onTriggered: startup = false
     }
 
+    function roleTexts(section) {
+       return [i18n("Connected"), i18n("Paired devices"), i18n("Nearby devices"), "-"][section]
```

Comment:
Please use a switch and the Enum values from the DeviceListModel, for example DeviceListModel.Connected

Reply:
Done